server.js

```diff
@@ -63,7 +63,7 @@ app.post("/books", async (request, response) => {
 app.delete("/books/:id/:email", async (request, response) => {
```

Comment:
Hi there!

Feedback here: Check this route, I am thinking this should be `app.delete('/books/:id', async (req, res) => {`